src/components/Calculator.js

```diff
@@ -1,71 +1,58 @@
 /* eslint-disable react/button-has-type */
 import './App.css';
-import React from 'react';
+import React, { useState } from 'react';
 import DigitButton from './component_mocks/DigitButton';
 import FunctionButton from './component_mocks/FunctionButton';
 import SpanTwo from './component_mocks/spanTwoButton';
 import Display from './component_mocks/Display';
 import OperationButton from './component_mocks/OperationButton';
 import calculate from '../logic/calculate';
 
-class App extends React.PureComponent {
-  constructor(props) {
-    super(props);
+function App() {
+  const [data, setData] = useState({
```

Comment:
- [ ] [_OPTIONAL_] It's best practice to name functions the same as the file name in this case` function Calculator`.